### PR TITLE
Wrap README to 80 characters where possible; minor rewording

### PR DESCRIPTION
**Pull request by jlw**
# Summary

While I was making a couple of other minor changes to the README, I noticed that large sections were wrapped to 80 characters and other sections were not. In case your desire as maintainers is to have the Markdown source, I've wrapped it everywhere except in long code samples and URLs.

I also made a couple minor content tweaks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 ### _extendable version manager_
 
-Supported languages include Ruby, Node.js, Elixir and [more](https://github.com/asdf-vm/asdf-plugins). Supporting a new language is as simple as [this plugin API](https://github.com/asdf-vm/asdf/blob/master/docs/creating-plugins.md).
+Supported languages include Ruby, Node.js, Elixir and [more][plugins].
+Supporting a new language is as simple as [this plugin API][plugin_api].
+
+[plugins]: https://github.com/asdf-vm/asdf-plugins
```
Comment (jlw) [inline README.md]: _using URL references to keep the line lengths short_

File: README.md [hunk 2]
```diff
@@ -147,7 +159,9 @@ asdf uninstall <name> <version>
 # asdf uninstall erlang 17.3
 ```
 
-_If a plugin supports downloading & compiling from source, you can also do this `ref:foo` (replace `foo` with the branch/tag/commit)._ You'll have to use the same name when uninstalling too.
+_If a plugin supports downloading & compiling from source, you can specify
+`ref:foo` where `foo` is a specific branch, tag, or commit._ You'll need to use
```
Comment (jlw) [inline README.md]: _the wording here seemed a bit awkward to me so I rewrote it a bit_

File: README.md [hunk 3]
```diff
@@ -68,15 +79,15 @@ Also if you're having issues with it not detecting the shims you've installed it
...
+specific tagged version in the GitHub branches/tags drop-down if you need to.
 
 ### Manage plugins
 
-Plugins are how asdf understands how to handle different packages.
-
-You can find a list of all asdf plugins in the [plugins repository](https://github.com/asdf-vm/asdf-plugins).
-
-There is a [super-simple API](https://github.com/asdf-vm/asdf/blob/master/docs/creating-plugins.md) for supporting more languages.
+Plugins are how asdf understands how to handle different packages. The [plugins
```
Comment (jlw) [inline README.md]: _This didn't seem like it should be 3 separate paragraphs, so I reworded it a bit and joined it into one._

**Comment by danhper** [comment]
Thanks, looks good!

**Pull request merged**